### DISCUSSION TITLE
Welcome to T-Display-S3-YT Discussions!

### DISCUSSION THREAD

**Discussion by nishad2m8** [nishad2m8/T-Display-S3-YT#4] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by johnnylowmango** [comment]
Hello there! You make some beautiful UI designs for these boards. My question is, how do I upload to the board using Platformio? I have copied "Libraries" folder and placed into the project Nixxie Clock lib folder, and I try to upload but it cant find "credentials.h"so it fails there.... I have looked for this file but cannot find it, any help is appreciated and thank you for showing us these awesome projects!

**Reply by johnnylowmango** [reply]
I've created a credential.h file but the display is just black backlit, it succeeds to upload though.

**Reply by johnnylowmango** [reply]
I have figured it out, for anyone else facing this issue, create "credential.h" file and type in your wifi info for it to boot.

**Reply by nishad2m8** [reply]
I forget to add in readme. happy to hear you figured out. ill update the readme. 

`credential.h`
```c++
#ifndef CREDENTIAL_H
#define CREDENTIAL_H

// Replace with your actual WiFi credentials
const char* ssid = "your_SSID";
const char* password = "your_PASSWORD";

#endif // CREDENTIAL_H
```

**Reply by AlHeeley** [reply]
does this file go into the lib folder? Im getting successful upload (trying to get the casio-style world clock working) but Lilygo t-display s3 remains black :(

**Reply by nishad2m8** [reply]
all the files from `00-❗libraries` copy to `lib` folder. don't change anything here. 

Also replace the `platformio.ini `entry with:  
```ini
platform = espressif32 @ 6.5.0
```

or use this code. i tested now. [PIO-Casio-4-12-25.zip](https://github.com/user-attachments/files/23926323/PIO-Casio-4-12-25.zip)

**Reply by AlHeeley** [reply]
Fantastic - that solved it! Many thanks - steep learning curve for me.

**Comment by AlHeeley** [comment]
Sorry - another basic question. Do you copy the files from the 00-! Libraries folder within Windows Explorer or cut/paste them in when you are in PlatformIO? Sometimes I'm not getting the option on the bottom bar to verify or upload the code?

**Reply by AlHeeley** [reply]
If its possible to post up a zip file of the moon clock code as you did for the casio world clock Id be most grateful!
